tests/vagrant/templates/Vagrantfile.tmpl

```diff
@@ -2,15 +2,15 @@
 # vim: set ft=ruby
 
 Vagrant.configure("2") do |config|
-    config.vm.box = "{{BOXNAME}}"
+    config.vm.box = "{BOXNAME}"
     # The nfs server on Leap has udp enabled by default, but not on Tumbleweed
     # Disable udp to make it working for any host system by default.
-    config.vm.synced_folder "{{ROOTDIR}}", "/srv/aquarium", type: "nfs", nfs_udp: false
+    config.vm.synced_folder "{ROOTDIR}", "/srv/aquarium", type: "nfs", nfs_udp: false
     config.vm.synced_folder ".", "/vagrant", disabled: true
 
     config.vm.define :"node01", primary: true do |node|
         node.vm.hostname = "node01"
-        node.vm.network "forwarded_port", guest: 1337, host: 1337, host_ip: "*"
+        node.vm.network "forwarded_port", guest: 1337, host: 1337
```

Comment:
why do you remove this?

Reply:
I didn't. must have been the move or something. I'll check it out.